regexes/device/mobiles.yml

```diff
@@ -30763,9 +30763,9 @@ Tele2:
   models:
     - regex: 'Tele2fon[_ ]V6'
       model: 'V6'
-    - regex: 'Tele2[ _](Mi[dn]i)[ _]1.1(?:[);/ ]|$)'
+    - regex: 'Tele2[ _](Mi[dn]i)[ _]1\.1(?:[);/ ]|$)'
       model: '$1 1.1'
-    - regex: 'Tele2[ _]Midi[ _]2.0(?:[);/ ]|$)'
+    - regex: 'Tele2[ _]Midi[ _]2\.0(?:[);/ ]|$)'
```

Comment:
```suggestion
    - regex: 'Tele2[ _]Midi[ _]2.0(?:[);/ ]|$)'
```

Reply:
This is matching `Tele2_Midi_2_0`, I'll make it a literal underscore.

Reply:
be1ae923